docs: toStringArray built-in function (#165)
reviewpad/docs: docs/guides/built-ins/functions/toStringArray/index.mdx

import CodeBlock from "@theme/CodeBlock";
import Example from "!!raw-loader!./example.yml";

**Description**:

> Converts a given JSON string into an array of strings or returns an error if it can not be converted.

Reviewer: ```suggestion
Converts a given JSON string into an array of strings. It returns an error the value can not be converted.
```
